services/datamanager/builtin/file_deletion.go

```diff
@@ -0,0 +1,137 @@
+package builtin
+
+import (
+	"context"
+	"errors"
+	"io/fs"
+	"math"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ricochet2200/go-disk-usage/du"
+
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/services/datamanager/datacapture"
+	"go.viam.com/rdk/services/datamanager/datasync"
+)
+
+// TODO change these values back to what they should be.
```

Comment:
Can remove! :) 

Could you post the results with the given tuning parameters into the PR description, and what you compared against?

Reply:
I posted what I have so far, lmk if theres something missing youd like me to add.